ios/VisionCameraPluginInatVision/VisionCameraPluginInatVisionModule.m

```diff
@@ -52,6 +54,68 @@ + (VCPVisionModel *)visionModelWithModelFile:(NSString *)modelPath {
     return cvModel;
 }
 
+- (MLMultiArray * _Nullable)combineVisionScores:(MLMultiArray *)visionScores with:(MLMultiArray *)geoScores error:(NSError **)error {
```

Comment:
@alexshepard for context we have two different modes of getting vision predictions, one is the frame processor and the other one being for a given image file. The methods for predicting on a given image file (uri) live in this module here, the frame processor lives in `ios/VisionCameraPluginInatVision/VisionCameraPluginInatVision.m`.

For enabling the geomodel for predicting from image file I have copied over those two functions here for combining vision and geo, and normalization over from the frane processor. I am sure there is a better way of avoiding code duplication but I don't know what it looks like in Obj-C. Could you make any suggestions on how to avoid duplicating any of the code I added in this file.